src/components/dialogmote/endre/EndreDialogmoteTekster.tsx

```diff
@@ -68,11 +82,13 @@ const EndreDialogmoteTekster = ({ dialogmote }: Props) => {
           generateArbeidstakerTidStedDocument(values)
         }
       />
+
```

Comment:
Fjern blank linje her

Reply:
Hvorfor det? Jeg tenker at vi kanskje heller burde legge til en blank linje mellom linje 101 og 102, så blir filen enklere å lese, siden de ulike delene er skilt litt fra hverandre? 🤷 (Evt. kunne man kanskje skilt ut FritekstSeksjon + Forhåndsvisning i en egen komponent, men det er jo i så fall en oppgave til senere 👍)

Reply:
Nevermind, jeg tenkte det ble rendret en blank linje, men det gjør det jo ikke. Da er det bare fint å ha det for lesbarheten sin skyld 👍